pkg/backend/openshiftcluster/adminupdate.go

```diff
@@ -6,14 +6,14 @@ package openshiftcluster
 import (
 	"context"
 
-	"github.com/Azure/ARO-RP/pkg/install"
+	"github.com/Azure/ARO-RP/pkg/cluster"
 )
 
 func (m *Manager) AdminUpdate(ctx context.Context) error {
 	// m.ocDynamicValidator.Dynamic is not called so that it doesn't block an
 	// admin update
 
-	i, err := install.NewInstaller(ctx, m.log, m.env, m.db, m.billing, m.doc, m.subscriptionDoc)
+	i, err := cluster.New(ctx, m.log, m.env, m.db, m.billing, m.doc, m.subscriptionDoc)
```

Comment:
I generally like just saying "New" instead of "NewX", like you've done. But here it almost implies a new cluster is necessarily created, when it's actually an update. Would `cluster.NewInstaller` be too weird?